applications/openshift/kubelet/kubelet_eviction_thresholds_set_hard_imagefs_available/tests/match.pass.sh

```diff
@@ -0,0 +1,459 @@
+#!/bin/bash
+# remediation = none
+
+kube_apipath="/kubernetes-api-resources"
```

Comment:
Thank you very much for adding these tests, this is what we should be doing for all rules. But, this rule seems to be failing:
```
ERROR - Script match.pass.sh using profile (all) found issue:
ERROR - Rule evaluation resulted in fail, instead of expected pass during initial stage 
ERROR - The initial scan failed for rule 'xccdf_org.ssgproject.content_rule_kubelet_eviction_thresholds_set_soft_imagefs_inodesfree'.
```

Reply:
found the issue! a upper case in yaml path